Website/src/rendering/src/assets/css/components/PageContent/Discover.css

```diff
@@ -0,0 +1,198 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap");
```

Comment:
Stylelint has found some issues in this PR. You can run stylelint with `jss stylelint` or `npm run stylelint`. You can also run eslint at the same time with `jss lint` or `npm run lint`.

```text
src/assets/css/components/PageContent/Discover.css
  30:16  ✖  Unexpected missing generic font family   font-family-no-missing-generic-family-keyword
 112:3   ✖  Unexpected duplicate "width"             declaration-block-no-duplicate-properties    
 113:3   ✖  Unexpected duplicate "top"               declaration-block-no-duplicate-properties    
 182:40  ✖  Unexpected "   "                         selector-descendant-combinator-no-non-space  
 183:3   ✖  Expected indentation of 0 spaces         indentation                                  
 183:16  ✖  Unexpected "   "                         selector-descendant-combinator-no-non-space  
 184:3   ✖  Expected indentation of 0 spaces         indentation                                  
 184:24  ✖  Unexpected "   "                         selector-descendant-combinator-no-non-space  
 185:3   ✖  Expected indentation of 0 spaces         indentation                                  
 192:40  ✖  Unexpected "   "                         selector-descendant-combinator-no-non-space  
 193:3   ✖  Expected indentation of 0 spaces         indentation                                  
 193:16  ✖  Unexpected "   "                         selector-descendant-combinator-no-non-space  
 194:3   ✖  Expected indentation of 0 spaces         indentation                                  
 194:27  ✖  Unexpected "   "                         selector-descendant-combinator-no-non-space  
 195:3   ✖  Expected indentation of 0 spaces         indentation

```